### PR TITLE
Change in scope of the index variable

### PR DESCRIPTION
**Pull request by sharukhsharu**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: lib/route.js [hunk 1]
```diff
@@ -128,10 +128,10 @@ function buildRouting (options) {
     const opts = { ...options }
 
     throwIfAlreadyStarted('Cannot add route when fastify instance is already started!')
-
+    // Validation for array of http Methods if the route has more than one
     if (Array.isArray(opts.method)) {
       /* eslint-disable no-var */
-      for (var i = 0; i < opts.method.length; i++) {
+      for (let i = 0; i < opts.method.length; i++) {
```
Comment (mcollina) [inline lib/route.js:134]: Please restore this, it was a `var` on purpose.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: lib/route.js (hunk 1 above)
Comment (sharukhsharu) [reply lib/route.js:134]: Sure, Just curious  was there any particular need for this reference

Comment (sharukhsharu) [reply lib/route.js:134]: I will close the PR.

Comment (mcollina) [reply lib/route.js:134]: using let in for loop is still slower than using `var`.

Comment (sharukhsharu) [reply lib/route.js:134]: oh sorry 😁 , i  forgot about the v8 perf diff with var and let

**Pull request closed without merging**